src/Appwrite/Auth/Phone.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Appwrite\Auth;
+
+use Appwrite\Extend\Exception;
+
+abstract class Phone
+{
+    /**
+     * @var string
+     */
+    protected string $user;
+
+    /**
+     * @var string
+     */
+    protected string $secret;
+
+    /**
+     * @param string $key
+     */
+    public function __construct(string $user, string $secret)
+    {
+        $this->user = $user;
+        $this->secret = $secret;
+    }
+
+    /**
+     * Send Message to phone.
+     * @param string $from
+     * @param string $to
+     * @param string $message
+     * @return void
+     */
+    abstract public function send(string $from, string $to, string $message): void;
+
+    /**
+     * @param string $method
+     * @param string $url
+     * @param array  $headers
+     * @param string $payload
+     *
+     * @return string
+     */
+    protected function request(string $method, string $url, array $headers = [], ?string $payload = null, ?string $userpwd = null): string
```

Comment:
We usually call this method `call`

Reply:
I got this from OAuth, I can change both either way.